docs/utils/masked/index.rst

```diff
@@ -137,7 +137,7 @@ In contrast, |Masked| is always wrapped around the data properly, i.e., a
 is never masked.  Indeed, one can see this from the class hierarchy::
 
   >>> mq.__class__.__mro__
-  (<class 'astropy.utils.masked.core.MaskedQuantity'>,
+  (<class 'abc.MaskedQuantity'>,
```

Comment:
Funny that this is changed! I guess it slightly obscures what is happening, but not a big deal.

Reply:
Yeah. I wouldn't mind figuring out how to fix the module. Some option in ``type``, maybe? I'm not seeing anything in the documentation.

Reply:
To me the best optional options are
1.  to set the module from the wrapped class, e.g. ``astropy.units.quantity.MaskedQuantity``.
2. to set the module from the calling file, e.g. ``docs.utils.masked.index.MaskedQuantity``.

Reply:
It looks like numpy has figured this out, but I can't track down how they do it. @mhvk, any insight?

Reply:
I vaguely recall that they explicitly set `__module__`, but I'm not sure...

Reply:
Not sure it is worth fretting too much about this - the classes are meant to be autogenerated, and while their initializers work, they can be counterintuitive (e.g., `MaskedNDArray` really uses the `ndarray` initializer).

Perhaps in that sense the name `abc....` is not so bad -- at least it is obvious it is not a regular class! One could actually envisage a name that passes on that it is autogenerated somehow...

Reply:
I've reverted to ``abc.<X>``.